### DISCUSSION TITLE
Welcome to wfdownloads Discussions!

### DISCUSSION THREAD

**Discussion by mambax7** [XoopsModules25x/wfdownloads#37] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by ggoffy** [comment]
Hi Michael

I have never used wfdownloads, therefore I have no experience. but I have translated language pack into german, provided it on myxoops and created an article https://www.myxoops.org/modules/publisher/index.php/item.146/

**Comment by ggoffy** [comment]
I made first checks and found no bugs :)

**Comment by mambax7** [comment]
Awesome!!! Thank you for the translations! 
There are still some open issues, and hopefully I'll be able to look at them this weekend.
